redact/v3/redact_requests.py

```diff
@@ -136,6 +137,11 @@ def post_job(
 
             return JobPostResponse(**response.json())
 
+    def _retrieve_file_name(self, headers: Dict[str, str]):
```

Comment:
should we extract that into a utility function?
the same function used in the v4 and they are pretty generic